api/include/mfxbrc.h

```diff
@@ -53,8 +53,19 @@ typedef struct {
 
 typedef struct {
     mfxI32 QpY;             // Frame-level Luma QP
+#if (MFX_VERSION >= 1029)
+    mfxU32 maxFrameSize;    // Max frame size in bytes (used for rePak)
+    mfxU16 maxNumRePak;     // Max number of rePak to provide MaxFrameSize (from 0 to 8)
+    mfxU8  deltaQP[8];      // deltaQP[i] is adding to QP value while i-rePak 
+    mfxU32 initial_cpb_removal_delay;
+    mfxU32 initial_cpb_removal_offset;
+    mfxU32 reserved1[6];
+    mfxU16 NumExtParam;
+    mfxExtBuffer** ExtParam;   // extention buffer list
```

Comment:
typo: extension

Reply:
Need to align style - mixing Camel, snake & Pascal notation in one struct.

Reply:
Also, different size under #if #else branches.
Supposed to be compensated for x64 by default 8-byte (ptr size) alignment, still a dangerous practice.
Need to combine 2 U16 and add extra reserved U32.
smth like this:
    mfxU32 MaxFrameSize;    // Max frame size in bytes (used for repack)
    mfxU8  DeltaQP[8];      // DeltaQP[i] is adding to QP value while i-repack 
    mfxU32 InitialCpbRemovalDelay;
    mfxU32 InitialCpbRemovalOffset;
    mfxU32 reserved[7];
    mfxU16 MaxNumRepack;     // Max number of repack to provide MaxFrameSize (from 0 to 8)
    mfxU16 NumExtParam;
    mfxExtBuffer** ExtParam;   // extension buffer list

Reply:
Fixed